content/tutorials/mapping.md

```diff
@@ -0,0 +1,103 @@
+---
+title: "Using SLAM for Map Building"
+date: 2021-03-20T17:15:01-07:00
+summary: "Learn how to create custom maps of your car's surroundings"
+difficulty: Intermediate
+duration: 0
+featured: true  # whether this is listed at / (must also be top 6 by weight)
+active: true    # whether this is listed at /tutorials/
+draft: false      # whether Hugo considers this a draft
+weight: 100
+---
+
+## By: Madison Doerr and Markus Schiffer
+
+### Introduction
+The MuSHr car operates in a mapped area to determine its position, orientation, and plans. We can create 
```

Comment:
Replace first sentence with: The MuSHR car uses a map of its environment to localize itself.